scripts/main/sidebar.js

```diff
@@ -107,7 +107,12 @@ sidebar.triggerSearch = function (search_string) {
 	lychee.goto("search/" + encodeURIComponent(search_string));
 };
 
-sidebar.toggle = function () {
+sidebar.userPreferenceVisibility = function () {
```

Comment:
I would change the method name too to match the session stored value.

Reply:
Yup, I was doing it as you typed :smile: thank you!